controllers/datamover.go

```diff
@@ -160,3 +180,149 @@ func (r *DPAReconciler) getDataMoverLabels() map[string]string {
 	labels[oadpv1alpha1.DataMoverDeploymentLabel] = "True"
 	return labels
 }
+
+// Check if there is a valid user supplied restic secret
+func (r *DPAReconciler) validateDataMoverCredential(resticsecret *corev1.Secret) bool {
+	if resticsecret == nil {
+		return false
+	}
+	for key, val := range resticsecret.Data {
+		if key == ResticPassword {
+			if len(val) != 0 {
+				return false
+			}
```

Comment:
should this be the other way around? returning true if there is a supplied password

Reply:
ha ha 100% :D what was I thinking. Thanks Emily